interface/reports/patient_edu_web_lookup.php

```diff
@@ -55,76 +44,51 @@ function searchResultsPopup(search_term,link) {
   }
 </script>
 
-<title><?php echo htmlspecialchars( xl('Find Patient Education Materials'), ENT_NOQUOTES); ?></title>
 </head>
 
-<body class="body_top">
-
-<span class='title'><?php echo htmlspecialchars( xl('Web Search'), ENT_NOQUOTES); ?> - <?php echo htmlspecialchars( xl('Patient Education Materials'), ENT_NOQUOTES); ?></span>
-
+<body class="body_top" onload="javascripts:document.forms[0].form_import_data.focus()">
+<div class="container">
+  <div class="row">
+<!-- -->
+    <div class='title'><?php echo  xlt('Web Search'); ?> - <?php echo xlt('Patient Education Materials'); ?></div>
```

Comment:
remove the 'title' class and place a page-header class. and wrap the text in h2 tags